Tracks: Fix CodeSniffer errors

psealock opened the pull request:
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

phpcs fixes for touched files in Tracks project.

psealock committed:
phpcs

psealock committed:
moar

psealock committed:
even more

psealock committed:
getting there

psealock committed:
dione

psealock committed:
admin

mikejolley commented on includes/admin/class-wc-admin-setup-wizard.php:
```diff
@@ -536,7 +536,7 @@ class="location-input wc-enhanced-select dropdown"
 		check_admin_referer( 'wc-setup' );
 
-		// phpcs:disable WordPress.VIP.SuperGlobalInputUsage.AccessDetected, WordPress.VIP.ValidatedSanitizedInput.InputNotValidated, WordPress.VIP.ValidatedSanitizedInput.MissingUnslash
+		// phpcs:disable WordPress.VIP.SuperGlobalInputUsage.AccessDetected, WordPress.VIP.ValidatedSanitizedInput.InputNotValidated, WordPress.VIP.ValidatedSanitizedInput.MissingUnslash, WordPress.Security.ValidatedSanitizedInput
```
I've just noticed all those disables... Maybe this block should be checked again. For example, unslashing is needed!

e.g.:

```
$address        = sanitize_text_field( $_POST['store_address'] );
```

to:

```
$address        = wc_clean( wp_unslash( $_POST['store_address'] ) );
```

Would you mind checking this? Also bare in mind the .VIP. rules no longer exist. I think it's .SECURITY. so they can be removed.

mikejolley commented on includes/admin/class-wc-admin.php:
```diff
@@ -203,29 +207,29 @@ public function prevent_admin_access() {
-			// print the preview email
-			echo $message;
+			// print the preview email.
+			echo esc_html( $message );
```
Might be html in here. Probably best to echo and add exclusion rule. It's not user generated.

mikejolley reviewed (changes requested):
Added some comments. The disables in wizard need a look.

psealock committed:
wc_clean

psealock committed:
dont escape

psealock commented:
Thanks @mikejolley !

I updated the preview emails `$message` to not escape html and removed the wizard disables in favour of `wc_clean` and `wp_unslash`. Thanks for the tip.
